build/release/containers.sh

```diff
@@ -4,28 +4,45 @@ scriptdir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 source ${scriptdir}/common.sh
 
 registry=quay.io/
-repo=rook/rookd
 
 build() {
     local type=$1
-    local os=$2
-    local arch=$3
-    local version=${RELEASE_VERSION}
-
```

Comment:
for readability I think we should leave these and not assume positional arguments to pass to build_artrifacts below